src/objects/Player.gd

```diff
@@ -25,8 +25,28 @@ func _unhandled_input(event: InputEvent) -> void:
 		return
 	if event is InputEventMouseMotion:
 		var mm_event := event as InputEventMouseMotion
-		rotation.y = wrapf(rotation.y - mm_event.relative.x * MOUSE_SENS.x, 0, TAU)
-		camera.rotation.x = clamp(camera.rotation.x - mm_event.relative.y * MOUSE_SENS.y, -PI / 2, PI / 2)
+		var relative := mm_event.relative
+
+		var window_size := OS.get_window_size()
+		var base_size := Vector2(
+				ProjectSettings.get_setting("display/window/size/width"),
+				ProjectSettings.get_setting("display/window/size/height")
+		)
+
+		# Because of the 2D scaling mode, the game "scales" our mouse input to match the current window size. That means
+		# if you make the window bigger, your mouse inputs will be relatively smaller. We don't want this, since that
+		# doesn't make sense for 3D mouse look. So here, we "un-scale" it back to normal
+		var scale := min(
```

Comment:
Why take the minimum as opposed to maximum? Does this mean sensitivity is scaled to the axis that was least stretched?

Reply:
Yeah I do that because that's how Godot scales the screen. You can see by resizing the window to extreme aspect ratios and looking at the UI